index.html

```diff
@@ -1135,6 +1160,169 @@ <h3>
+        such as on a home-screen, in an app store, or even on top of
+        notifications.
+      </p>
+      <p>
+        For this reason, image can be declared with different sizes and for
+        different purposes. A user agent can then use these data to decide
+        which icons to use in different situations. For instance, a user agent
+        may prefer showing icons with the <a data-lt="badge purpose">badge</a>
+        purpose on top of notifications, as <a data-lt=
+        "badge purpose">badge</a> indicates an icon that is can be placed on
+        top of other content, in size constrained situations.
+      </p>
+      <section>
+        <h2>
+          Icon masks and safe zone
+        </h2>
+        <p>
+          Some platforms have their own preferred icon shape, but as web
+          applications should work across multiple platforms, it is possible to
+          indicate that an icon may be supercomposed with a mask, by adding the
+          <a data-lt="masking purpose">masking</a> purpose. This allows the
+          platform to ensure that the icon looks well integrated with the
+          platform, and even apply different masks and background colors in
+          different places throughout the platform.
+        </p>
+        <p>
+          Any icon indicating that it can be supercomposed with a mask, much
+          make sure that all important parts are within the <dfn>safe
+          zone</dfn>, which is defined as a circle with center point in the
+          center of the icon and with a radius of 2/5 (40%) of the icon size,
+          which means the smaller of the icon width and height, in case the
+          icon is not retangular.
+        </p>
+        <figure>
+          <img src="images/safe-zone.svg" width="340" height="340" alt=
+          "safe zone illustrated">
+          <figcaption>
+            The safe zone is a centrally positioned circle, with radius 2/5
+            (40%) of the icon size.
+          </figcaption>
+        </figure>
+        <p>
+          All pixels in this zone are guaranteed to be seen in all masks. What
+          is outside may be visible, but are not guaranteed to be so.
+        </p>
+        <p>
+          The user agent MAY apply a mask of any size, making any pixels that
+          are more than 2/5ths of the image size (minimum of width and height
+          if non-square) away from the center (the <a>safe zone</a>)
+          transparent.
+        </p>
+        <p>
+          The user agent MUST NOT make any pixel within the safe zone
+          transparent.
+        </p>
+        <p>
+          The user agent MAY superimpose the icon on top of a colored
+          background of any kind and MAY further enlarge the icon by adding
+          additional padding.
+        </p>
+        <section>
+          <h3>
+            Examples of masks
+          </h3>
+          <p class="note">
+            By staying inside the <a>safe zone</a>, most icons will have around
```

Comment:
This should all be moved to MDN instead, IMO.

Reply:
Can we remove when things have been merged and that has been added to MDN?

Reply:
Sure. That would be fine.